Enterprise-CMCS/macpro-mdct-mcr
# MDCT-2301 - statusing ETL script

## Summary
[MDCT-2301](https://qmacbis.atlassian.net/browse/MDCT-2301)

To faciliate the ability to have some entities be optional for sake of form completeness, a new object has been added to mcpar.json. This new object will need to be added to the existing formTemplates for all existing MCPAR reports

Acceptance Criteria
At the root node of each formTemplate, add the following object:

  "entities": {
    "sanctions": { "required": false },
    "accessMeasures": { "required": true },
    "qualityMeasures": { "required": true },
    "plans": { "required": true },
    "bssEntities": { "required": true }
  }
Dev Note: 
This should be done using ETL, similar to the way we did the ETL for Winter Storm updates.

### Description


### Related ticket


### How to test

I think we'll need to test this against one of the test environments with existing data
1. in an environment with existing form templates that don't have the `entities` object, run the script
2. verify all form templates now have correct `entities` object 

### Important updates


updated [services/app-api/serverless.yml](https://github.com/Enterprise-CMCS/macpro-mdct-mcr/pull/11120/files#diff-ee790d511870504b4f2e29c38bb1f7f4e22ab59f84ea337b20159b0dfb186f39) to include new script

### Author checklist

- [ x] I have performed a self-review of my code
- [ x] I have added [thorough](https://bit.ly/3zPrxuZ) tests, if necessary
